app/containers/Dialog.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, Alert } from 'react-native';
+import RNDialog from 'react-native-dialog';
+
+import EventEmitter from '../lib/methods/helpers/events';
+import { Services } from '../lib/services';
+import I18n from '../i18n';
+import log, { logEvent } from '../lib/methods/helpers/log';
+import events from '../lib/methods/helpers/log/events';
+import { useTheme } from '../theme';
+import sharedStyles from '../views/Styles';
+
+const styles = StyleSheet.create({
+	title: {
+		fontSize: 18,
+		...sharedStyles.textMedium
+	},
+	description: {
+		fontSize: 15,
+		...sharedStyles.textRegular
+	},
+	buttonText: {
+		fontSize: 14,
+		...sharedStyles.textSemibold
+	}
+});
+
+export const LISTENER_DIALOG = 'Dialog';
+
+let listener: Function;
+
+interface IMessage {
+	title: string;
+	description: string;
+	inputLabel: string;
+	data: any;
+}
+
+const Dialog = (): React.ReactElement => {
+	const { colors } = useTheme();
+
+	const [isVisible, setIsVisible] = useState(false);
+	const [text, setText] = useState('');
```

Comment:
why you need thys text?

Reply:
Sir this is to store user's input text for report